pkg/d1to2fix.pkg

```diff
@@ -1,5 +1,5 @@
-OPTS = dict(
-    name = VAR.name,
+OPTS.update(
```

Comment:
Shouldn't this adaptation be done when the MakD submodule is advanced?

Reply:
Yes, but that is already merged :shame: Travis script wasn't generating any packages so did not notice makd upgrade has broken it.